container.go

```diff
@@ -188,7 +190,17 @@ func (c *ContainerRequest) GetContext() (io.Reader, error) {
 	}
 	c.Context = abs
 
-	buildContext, err := archive.TarWithOptions(c.Context, &archive.TarOptions{})
+	// based on https://github.com/docker/cli/blob/master/cli/command/image/build/dockerignore.go#L14
+	var excluded []string
+	if f, err := os.Open(filepath.Join(abs, ".dockerignore")); err != nil {
+		var err error
+		excluded, err = ignorefile.ReadAll(f)
+		if err != nil {
+			return nil, fmt.Errorf("error reading .dockerignore: %w", err)
```

Comment:
Do we want to raise an error here and fail our logic, or simply log the error and continue?

Reply:
At this point in the code the dockerignore file is read, and any errors are coming from the parsing of the file. I feel like if this ever fails its better to return the error and let the user fix the problem.